src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Methodologies/MethodologyController.cs

```diff
@@ -10,41 +13,46 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.Metho
 {
     [Authorize]
     [ApiController]
+    [Route("api")]
     public class MethodologyController : ControllerBase
     {
         private readonly IMethodologyService _methodologyService;
         private readonly IMethodologyAmendmentService _methodologyAmendmentService;
 
         public MethodologyController(
-            IMethodologyService methodologyService, 
+            IMethodologyService methodologyService,
             IMethodologyAmendmentService methodologyAmendmentService)
         {
             _methodologyService = methodologyService;
             _methodologyAmendmentService = methodologyAmendmentService;
         }
 
-        [Produces("application/json")]
-        [ProducesResponseType(typeof(MethodologySummaryViewModel), 200)]
-        [ProducesResponseType(typeof(ValidationProblemDetails), 400)]
-        [HttpPost("api/publication/{publicationId}/methodology")]
+        [HttpPost("publication/{publicationId}/methodology")]
         public Task<ActionResult<MethodologySummaryViewModel>> CreateMethodology(Guid publicationId)
         {
             return _methodologyService
                 .CreateMethodology(publicationId)
                 .HandleFailuresOrOk();
         }
-        
-        [Produces("application/json")]
```

Comment:
I think these were here for Swagger to use for generating its doco.  Don't know if it's worth keeping them in or not though, as we certainly aren't consistent with it.

Reply:
I saw that we weren't adding them to newer stuff so decided to get rid of them here rather than spend time maintaining them. I will bring it up to discuss with the team.